sql/migrations/20220224185216_world.sql

```diff
@@ -0,0 +1,2258 @@
+(54752, 27, 2303.39, -5376.1, 83.8028, 100),
+(54752, 28, 2309.54, -5371.37, 85.9728, 100),
+(54752, 29, 2315.65, -5361.15, 87.6213, 100),
+(54752, 30, 2319.76, -5354.2, 87.3382, 100),
+(54752, 31, 2315.91, -5354.27, 87.7765, 100),
+(54752, 32, 2320.69, -5347.79, 87.5548, 100),
+(54752, 33, 2325.82, -5334.41, 82.0919, 100),
+(54752, 34, 2332.27, -5324.26, 82.0671, 100),
+(54752, 35, 2333.67, -5312.79, 82.0739, 100);
+
+DELETE FROM `creature_movement` WHERE `id` = 53866;
+UPDATE `creature` SET `movement_type` = 0 WHERE `guid` = 53866;
+INSERT INTO `creature_groups` (`leader_guid`, `member_guid`, `dist`, `angle`, `flags`) VALUES 
+(53865, 53865, 0, 6.25244, 11),
+(53865, 53866, 3, 1.57, 11);
+
+
+DELETE FROM `creature_movement` WHERE `id` = 53865;
+INSERT INTO `creature_movement` (`id`, `point`, `position_x`, `position_y`, `position_z`, `orientation`) VALUES 
+(53865, 1, 2280.11, -5293.31, 83.956, 100),
+(53865, 2, 2279.3, -5281.28, 82.4405, 100),
+(53865, 3, 2278.77, -5266.62, 81.4722, 100),
+(53865, 4, 2279.01, -5274.7, 82.0752, 100);
+
+-- ----------------------------------------------------------------------
+
+-- Forman Bonds Should Talk When Summoning Guards
+DELETE FROM `creature_ai_scripts` WHERE `id`=230503;
+INSERT INTO `creature_ai_scripts` (`id`, `delay`, `command`, `datalong`, `datalong2`, `datalong3`, `datalong4`, `target_param1`, `target_param2`, `target_type`, `data_flags`, `dataint`, `dataint2`, `dataint3`, `dataint4`, `x`, `y`, `z`, `o`, `condition_id`, `comments`) VALUES (230503, 0, 10, 7360, 300000, 0, 0, 0, 0, 0, 0, 0, 0, 1, 1, 0, 0, 0, 0, 0, 'Foreman Bonds - Summon Creature Dun Garok Soldier');
+INSERT INTO `creature_ai_scripts` (`id`, `delay`, `command`, `datalong`, `datalong2`, `datalong3`, `datalong4`, `target_param1`, `target_param2`, `target_type`, `data_flags`, `dataint`, `dataint2`, `dataint3`, `dataint4`, `x`, `y`, `z`, `o`, `condition_id`, `comments`) VALUES (230503, 0, 10, 7360, 300000, 0, 0, 0, 0, 0, 0, 0, 0, 1, 1, 0, 0, 0, 0, 0, 'Foreman Bonds - Summon Creature Dun Garok Soldier');
+INSERT INTO `creature_ai_scripts` (`id`, `delay`, `command`, `datalong`, `datalong2`, `datalong3`, `datalong4`, `target_param1`, `target_param2`, `target_type`, `data_flags`, `dataint`, `dataint2`, `dataint3`, `dataint4`, `x`, `y`, `z`, `o`, `condition_id`, `comments`) VALUES (230503, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 609, 0, 0, 0, 0, 0, 0, 0, 0, 'Foreman Bonds - Talk');
+
+-- ----------------------------------------------------------------------
+
+-- Update A-Me 01 Flags
+UPDATE `creature_template` SET `unit_flags` = 768 WHERE `entry` = 9623;
+
+-- ----------------------------------------------------------------------
+
+-- Remove Creature Under Map
+DELETE FROM `creature` WHERE `guid` = 50190;
+
+-- ----------------------------------------------------------------------
+
+-- Set Correct Start Date For Southshore Assassins
+UPDATE `game_event` SET `start_time` = '2022-01-01 00:19:48' WHERE `entry`=152;

```

Comment:
I really doubt it would be at 00:19:48, it's prolly a between time value

Reply:
I thought this was just an arbitrary value that enables the event?